tests/bluetooth/tester/src/btp/btp_bap.h

```diff
@@ -109,6 +111,69 @@ struct btp_bap_broadcast_sink_stop_cmd {
 	uint8_t broadcast_id[3];
 } __packed;
 
+#define BTP_BAP_BROADCAST_SINK_BIS_SYNC		0x10
+struct btp_bap_broadcast_sink_bis_sync_cmd {
+	bt_addr_le_t address;
+	uint8_t broadcast_id[3];
+	uint32_t requested_bis_sync;
+} __packed;
+
+#define BTP_BAP_DISCOVER_SCAN_DELEGATORS	0x11
+struct btp_bap_discover_scan_delegators_cmd {
+	bt_addr_le_t address;
+} __packed;
+
+#define BTP_BAP_BROADCAST_ASSISTANT_SCAN_START	0x12
+struct btp_bap_broadcast_assistant_scan_start_cmd {
+	bt_addr_le_t address;
+} __packed;
+
+#define BTP_BAP_BROADCAST_ASSISTANT_SCAN_STOP	0x13
+struct btp_bap_broadcast_assistant_scan_stop_cmd {
+	bt_addr_le_t address;
+} __packed;
+
+#define BTP_BAP_ADD_BROADCAST_SRC		0x14
+struct btp_bap_add_broadcast_src_cmd {
+	bt_addr_le_t address;
+	bt_addr_le_t broadcaster_address;
+	uint8_t advertiser_sid;
+	uint8_t broadcast_id[3];
+	uint8_t padv_sync;
+	uint16_t padv_interval;
+	uint8_t num_subgroups;
+	uint32_t bis_sync;
+} __packed;
+
+#define BTP_BAP_REMOVE_BROADCAST_SRC		0x15
+struct btp_bap_remove_broadcast_src_cmd {
+	bt_addr_le_t address;
+	uint8_t src_id;
+} __packed;
+
+#define BTP_BAP_MODIFY_BROADCAST_SRC		0x16
+struct btp_bap_modify_broadcast_src_cmd {
+	bt_addr_le_t address;
+	uint8_t src_id;
+	uint8_t padv_sync;
+	uint16_t padv_interval;
+	uint8_t num_subgroups;
+	uint32_t bis_sync;
+} __packed;
+
+#define BTP_BAP_SET_BROADCAST_CODE		0x17
+struct btp_bap_set_broadcast_code_cmd {
+	bt_addr_le_t address;
+	uint8_t src_id;
+	uint8_t broadcast_code[16];
```

Comment:
Can consider using `BT_AUDIO_BROADCAST_CODE_SIZE` instead of `16` for the broadcast codes